### ISSUE TITLE
Rename `userDBs` to avoid confusing with `userDB`

### ISSUE THREAD

**Issue by BigBlueHat** [colinskow/superlogin#97]
`privateDBs` or `personalDBs` might be clearer.

`userDBs` and `userDB` are very different things and despite their differing location in the config JSON, it's tricky learning about how to use each when they have nearly the same name.

Thanks!

**Comment by colinskow** [comment]
I agree with you, but it would also be a breaking change that would require deprecation notices. There is already a comment in the `config.example.js` which makes it clear: `// Custom settings to manage personal databases for your users`.

If I could go back in time I would choose a different name, but at this point I don't think the disruption to current users would be worth it. I'll keep this on file for the next breaking changes release.

**Comment by BigBlueHat** [comment]
How about promoting `privateDBs` (or whatever), but supporting the old variable name and encouraging moving to the new name?
